README.md

```diff
@@ -176,6 +233,7 @@ export const App = () => {
   bottom?: string;
   left?: string;
   } | undefined }: The margin is used to set the separation between the items. You can use different margin for the first and last item to get better results.
+- `ref` { React.RefObject\<HTMLDivElement\> | undefined }. You can use it to programatically focus the `SnapItem` for example.
```

Comment:
If the dev can use the ref to his element inside the children we can avoid this.

Reply:
Updated!